README.md

```diff
@@ -80,11 +80,39 @@ Read the [Getting started page](https://flo-scss.flo.center/docs/getting-started
 
 Documentation search is powered by [Algolia's DocSearch](https://community.algolia.com/docsearch/). Working on our search? Be sure to set `debug: true` in `site/docs/assets/js/src/search.js` file.
 
+
 ## Contributing
 
 Please read through our [contributing guidelines](https://github.com/twbs/bootstrap/blob/master/CONTRIBUTING.md). Included are directions for opening issues, coding standards, and notes on development.
 
 
+### SVG
+
+#### Formatting a new SVG
+
+Each icon in the system is a formatted svg with a `viewBox="0 0 500 500"`, a path fill="currentColor" and a descriptive class name.
+
+The SVG is created on a 500px x 500px art board. SVG's created in a design program such as Adobe Illustrator or Sketch. Clean up an svg with this online tool https://jakearchibald.github.io/svgomg/. Make sure no extra `g` `tags`, `masks`, or `nested paths` are present.
+
+Format Guidelines
+* `fill=currentColor` is set
+* `viewBox="0 0 500 500"` is present
+* no inline width, height, or colors
+* use of classes to reference the SVG
+* the svg is flat with only a path tag. No `g` `tags`, `masks`, or `nested paths`
+
```

Comment:
Lines 93-109 I think should be in the icons.md file as those guidelines/rules pertain to best practices for all svgs, not just for the ones uploaded to the docs.

Reply:
ok